src/test/ui/consts/const-eval/union-ice.stderr

```diff
@@ -27,7 +27,7 @@ LL | |         unsafe { UNION.field3 },
 ...  |
 LL | |     a: 42,
 LL | | };
-   | |__^ type validation failed: encountered undefined bytes at .b[1]
+   | |__^ type validation failed: encountered uninitialized value at .b[1]
```

Comment:
Is this really the only test where we ensure that the index computation in the fast path is working properly?

Reply:
Yea, I don't remember having a dedicated test for this. With validation I think that we have a bunch of untested cases.